modules/agent-installer-architectures.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="agent-install-networking_{context}"]
+= Optional: Verifying the architecture of an Agent-based installer cluster
+
+After installing an {product-title} cluster using the Agent-based Installer, you can check the architecture of the cluster.
+
+.Prerequisites: 
+
+* You installed the {oc-first}.
+* You have the `openshift-install` binary.
+
+.Procedure: 
+
+. Log in to the {oc-first}.
+
+. Check your release payload by running the following command: 
+[source,terminal]
++
+----
+$ ./openshift-install version
+----
++
+.Example output 
+[source,terminal]
+----
+./openshift-install 4.16.z
```

Comment:
Does the user output actually show "z" for the z-stream, or should there be a true release number in this example, such as `4.16.0`?

Reply:
oh, right. it will show the true release number, let's write here `4.16.0`

Reply:
Updated as per the suggestion.